dotnetapp-4.7/Dockerfile

```diff
@@ -1,4 +1,10 @@
+FROM microsoft/dotnet-framework-build:4.7.1 AS build-env
```

Comment:
I'm thinking a `.dockerignore` should be created next to the Dockerfile to ignore any `bin` and `obj` folders in the case where the project may have been built locally.